src/Opdex.Platform.WebApi/Models/Responses/OHLC/OhlcDecimalResponseModel.cs

```diff
@@ -12,31 +12,27 @@ public class OhlcDecimalResponseModel
     /// Value at the start of the time interval.
     /// </summary>
     /// <example>55.00</example>
-    [NotNull]
     [Range(0, double.MaxValue)]
     public decimal Open { get; set; }
 
     /// <summary>
     /// Highest value during the time interval.
     /// </summary>
     /// <example>55.55</example>
-    [NotNull]
     [Range(0, double.MaxValue)]
```

Comment:
Just curious, are these ranges still needed?

Reply:
You're right they're not needed, removed